### ISSUE TITLE
Bitget cant subcribe WS channel orders and ordersAlgo channels at same time

### ISSUE THREAD

**Issue by dmunozfer** [ccxt/ccxt#17916]
### Operating System

Windows

### CCXT Version

3.0.103

### Description

@carlosmiei Related to https://github.com/ccxt/ccxt/pull/17906

It is not possible to subscribe to both channels "orders" and "ordersAlgo" at the same time. I think it has something to do with the messageHash and subscriptionHash parameters, but I don't know how to fix it.

### Code

```
  

```

**Comment by carlosmiei** [comment]
@dmunozfer Yeah most likely, will check it soon. Thanks for letting me know.

**Comment by dmunozfer** [comment]
> 

 I just wanted to follow up on the issue I reported earlier. Have there been any updates or progress on resolving it? I appreciate your attention to this matter and look forward to hearing from you. Thank you again for your assistance!

**Comment by ttodua** [comment]
old topic, but after checking this, i found that it is now supported. you have just to pass `trigger: true` in parameters: (specifically for bitget)
```
await ex.watchOrders (symbol);
await ex.watchOrders (symbol, {trigger:true});
```

let us know if anyone has any issues
